package.json

```diff
@@ -46,24 +46,24 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@cucumber/cucumber": "^8.3.0",
-    "@cucumber/cucumber-expressions": "^15.2.0",
+    "@cucumber/cucumber": "^8.5.1",
+    "@cucumber/cucumber-expressions": "^16.0.0",
     "@cucumber/gherkin": "^24.0.0",
     "@cucumber/gherkin-streams": "^5.0.1",
     "@cucumber/message-streams": "^4.0.1",
-    "@cucumber/messages": "^19.0.0"
+    "@cucumber/messages": "^19.1.2"
   },
   "peerDependencies": {
-    "@cucumber/cucumber": "^8.0.0",
-    "@cucumber/cucumber-expressions": "^15.0.0",
-    "testcafe": "~1.18.0"
+    "@cucumber/cucumber": "^8.5.1",
```

Comment:
No need to change this, any version 8.X.Y should work just fine because the guy working on cucumber use standard versionning too.
"8.0.0" => that specific version
"~8.0.0" => version 8.0.X (any that matches that pattern)
"^8.0.0" => version 8.X.Y

The whole point of setting it as ^8.0.0 was to allow the users of this package to use whatever version 8 they want.